Speeding up radsw

YakelynRJ opened the pull request:
The goal of this PR is to jit spcvrtm. It used to take 23.9 seconds and now it takes 0.1 seconds. 
This PR, 
1. Removed the self parameter from spcvrtm. 
2. Given that spcvrtm used vrtqdr that also contained self, I extracted vrtqdr and spcvrtm outside the RadSWClass to be called without the self parameter and I used @jit without the @staticmethod 

Coverage reports (updated automatically):
- test_unit: [83%](https:\/\/output.circle-artifacts.com\/output\/job\/d3b3bb0f-1654-4b43-b805-361646e5ef3c\/artifacts\/0\/tmp\/coverage\/htmlcov-test_unit\/index.html)

nbren12 reviewed (approved):
Thanks! Amazing speedups from jit. I agree the function probably had to be moved.

YakelynRJ commented:
The goal of this PR is to jit spcvrtm. It used to take 23.9 seconds and now it takes 0.1 seconds.
This PR,

Removed the self parameter from spcvrtm.
Given that spcvrtm used vrtqdr that also contained self, I extracted vrtqdr and spcvrtm outside the RadSWClass to be called without the self parameter and I used @jit without the @staticmethod
